### DISCUSSION TITLE
How to fetch labels for a merge request using the package?

### DISCUSSION THREAD

**Discussion by fervand1** [python-gitlab/python-gitlab#2758] [General]
Hello, Using the package I can we can get merge request metadata like title, description with its id but I couldn't get the labels. Is there a way we can get labels associated to a merge request?

**Comment by JohnVillalovos** [comment]
As far as I can tell they should be returned when you get a merge request.

https://docs.gitlab.com/ee/api/merge_requests.html#get-single-mr

Can you give some example code of what you are doing that isn't working?

**Comment by fervand1** [comment]
@JohnVillalovos Sorry, it is working fine. I was trying to fetch labels associated with a merge request like below.
![image](https://github.com/python-gitlab/python-gitlab/assets/44014460/dca1daae-152d-468b-a144-4525d8c7450b)

I was accessing wrong attribute "label" instead of "labels".
Thank you for assisting

**Reply by JohnVillalovos** [reply]
You're welcome. Glad to hear you figured it out.
